spec/helpers/social_link_helper_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+describe SocialLinkHelper do
+  let_it_be_readonly(:user) { create(:user) }
+
+  describe ".user_twitter_link" do
+    let(:value) { helper.user_twitter_link(user) }
+
+    context "when user has username" do
+      it "returns link with handler" do
+        expect(value).to include("@#{user.twitter_username}")
+      end
+
+      it "returns link to user" do
+        expect(value).to include("https://twitter.com/#{user.twitter_username}")
+      end
+    end
+
+    context "when user doesn\'t have a username" do
```

Comment:
Minor detail: no need to escape the apostrophe here; the double-quotes around the phrase ensure that it will be parsed correctly 😄

Reply:
My bad. I usually work with single quotes and I tend to escape things. Thanks for the sharp eye.